cognite/neat/rules/models/dms/_rules_input.py

```diff
@@ -146,7 +146,9 @@ def dump(self, default_space: str, default_version: str) -> dict[str, Any]:
             "Value Type": value_type,
             "Property (linage)": self.property_ or self.view_property,
             "Class (linage)": (
-                ClassEntity.load(self.class_, prefix=default_space, version=default_version) if self.class_ else None
+                ClassEntity.load(self.class_ or self.view, prefix=default_space, version=default_version)
+                if self.class_ or self.view
+                else None
```

Comment:
I think this was a bug in the `DMSPropertyInput`. We should default to use the view if the class is not set.